release-notes/10.0/preview/preview1/README.md

```diff
@@ -0,0 +1,36 @@
+# .NET 10 Preview 1 - Release Notes
+
+.NET 10 Preview 1 released on February X, 2025. Find more information on new features released in .NET 10 Preview 1 by browsing through the release notes below:

```

Comment:
Is it necessary to include the date here? We didn't for [.NET 9 preview release notes](https://github.com/dotnet/core/blob/dotnet10p1/release-notes/9.0/preview/preview1/9.0.0-preview.1.md).

Reply:
I think we did it eventually, although maybe not necessary?